.github/ISSUE_TEMPLATE/infrastructure-request-form-default.yml

```diff
@@ -0,0 +1,92 @@
+labels: ["infrastructure-request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Request new infrastructure or change an existing configuration. _All answers are optional._ The Platform Infrastructure team will work with you to gather all necessary requirements before processing your request. Providing information upfront can help to expedite your request. 
+  - type: input
+    id: team-name
+    attributes:
+      label: Team name
+      description: What is your team's name?
+      placeholder: e.g. Infrastructure Team
+    validations:
+      required: false
+  - type: input
+    id: application-name
+    attributes:
+      label: Application name
+      description: What is the name of your applicaiton or service?  
+      placeholder: e.g. Vets-API _or_ 1095-B
+    validations:
+      required: false
+  - type: input
+    id: application-repository
+    attributes:
+      label: Software repository
+      description: Please provide a link to your application's source-code repository  
+      placeholder: e.g. https://github.com/department-of-veterans-affairs/<my-repo>
+    validations:
+      required: false
+  - type: input
+    id: product-owner
+    attributes:
+      label: Product owner
+      description: Who is your team's product owner?
+      placeholder: e.g. Saima Simran
+    validations:
+      required: false
+  - type: input
+    id: product-manager
+    attributes:
+      label: Product manager
+      description: Who is your team's product manager?
+      placeholder: e.g. Hannah Ilja
+    validations:
+      required: false
+  - type: input
+    id: related-issues
+    attributes:
+      label: Link to initiative or epic 
+      description: Please share related issue(s) for additional context   
+      placeholder: e.g. https://github.com/department-of-veterans-affairs/va.gov-team/issues/<my-issue>
+    validations:
+      required: false
+  - type: textarea
+    id: infrastructure-needed
+    attributes:
+      label: Infrastructure needed 
+      description: Please describe your infrastructure needs.  
+      placeholder: e.g. https://github.com/department-of-veterans-affairs/va.gov-team/issues/<my-issue>
+    validations:
+      required: false
+  - type: dropdown
+    id: request-type
+    attributes:
+      label: New or existing 
+      description: Are you requesting to set up new infrastructure or modify an existing configuration?
+      options:
+        - New
+        - Existing
+        - Unsure
+    validations:
+      required: false
+  - type: textarea
+    id: service-accounts
+    attributes:
+      label: Users and service accounts 
+      description: If new, please provide a list of users and/or service accounts that need access to this infrastructure.  
+      placeholder: e.g. https://github.com/department-of-veterans-affairs/va.gov-team/issues/<my-issue>
+    validations:
+      required: false
+  - type: checkboxes
```

Comment:
Is it possible to do a horizontal line above the Acceptance Criteria section, or is that only possible with markdown? I was thinking it might be nice to visually separate the section we're responsible for from the main section.

Reply:
I was thinking that as well. I can look at bit more closely at the docs